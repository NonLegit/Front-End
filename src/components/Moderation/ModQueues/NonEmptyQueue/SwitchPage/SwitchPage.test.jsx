import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import SwitchPage from './SwitchPage';

test('SwitchPage test snapshot', async () => {
  const tree = renderer.create(
    <SwitchPage />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
