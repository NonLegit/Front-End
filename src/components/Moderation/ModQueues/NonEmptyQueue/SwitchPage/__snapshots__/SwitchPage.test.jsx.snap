// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SwitchPage test snapshot 1`] = `
<div
  className="MuiBox-root css-1yzmdw6"
>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-hr53ul-MuiButtonBase-root-MuiButton-root"
    disabled={false}
    onBlur={[Function]}
    onContextMenu={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    Back
  </button>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-hr53ul-MuiButtonBase-root-MuiButton-root"
    disabled={false}
    onBlur={[Function]}
    onContextMenu={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    Next
  </button>
</div>
`;
